Tests/js/views/assets/ez-locationviewview-tests.js

```diff
@@ -314,6 +314,15 @@ YUI.add('ez-locationviewview-tests', function (Y) {
                 }
             });
 
+            Y.eZ.LocationViewVersionsTabView = Y.Base.create('locationViewVersionsTabView', Y.View, [], {}, {
+                ATTRS: {
+                    content: {},
+                    contentType: {},
+                    config: {},
+                    priority: {},
+                }
+            });
+
```

Comment:
in addition to that, I think you also have to add some tests to check that the versions tab view receives the expected parameters.